### PR TITLE
[BPK-2813]: Updating gray colors

### PR DESCRIPTION
**Pull request by ojcurtis**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-tokens/src/common/base/colors.json [hunk 1]
```diff
@@ -11,16 +11,16 @@
...
-    "GRAY_300": "#B2AEBD",
-    "GRAY_400": "#9A95A7",
-    "GRAY_500": "#817B8F",
+    "GRAY_50": "#F1F2F8",
+    "GRAY_100": "#DDDDE5",
+    "GRAY_200": "#CDCDD7",
+    "GRAY_300": "#B2B2BF",
+    "GRAY_400": "#8f90a0",
+    "GRAY_500": "#68697F",
     "GRAY_600": "#696179",
```
Comment (georgegillams) [inline packages/bpk-tokens/src/common/base/colors.json]: I'm actually now thinking maybe we should make this the average of 500 and 700 🤔 
```suggestion
     "GRAY_600": "#565770",
```

File: packages/bpk-tokens/src/common/base/colors.json [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
-    "GRAY_700": "#524C61",
+    "GRAY_700": "#444560",
     "GRAY_800": "#3B344B",
```
Comment (georgegillams) [inline packages/bpk-tokens/src/common/base/colors.json]: As above
```suggestion
     "GRAY_800": "#2B2C4B",
```

File: packages/bpk-tokens/src/common/base/colors.json (hunk 1 above)
Comment (ojcurtis) [reply packages/bpk-tokens/src/common/base/colors.json]: I would suggest we leave it as they are, as this should hopefully mean consumers are forced to change the usage from deprecated values to non deprecated values 😄

File: packages/bpk-tokens/src/common/base/colors.json (hunk 2 above)
Comment (ojcurtis) [reply packages/bpk-tokens/src/common/base/colors.json]: I would suggest we leave it as they are, as this should hopefully mean consumers are forced to change the usage from deprecated values to non deprecated values 😄

**Pull request merged**